met-web/src/components/userManagement/userDetails/AddToEngagement.tsx

```diff
@@ -36,6 +36,7 @@ import { HTTP_STATUS_CODES } from 'constants/httpResponseCodes';
 export const AddToEngagementModal = () => {
     const { savedUser, addUserModalOpen, setAddUserModalOpen, getUserMemberships, getUserDetails } =
         useContext(UserDetailsContext);
+    console.log('user', savedUser);
```

Comment:
We can remove the logging for now. I should mentioned some calls were still present in the code

Reply:
removed